src/act.wizard.c

```diff
@@ -2632,16 +2640,39 @@ void do_start(struct char_data *ch) {
    */
   if ((r_num = real_object(12)) >= 0) {
     obj = read_object(r_num, REAL);
-    obj_to_char(obj, ch);       /* bread   */
-    obj = read_object(r_num, REAL);
-    obj_to_char(obj, ch);       /* bread   */
+    if (!obj) {
+      send_to_char("Game error loading object. File a bug.\n\r", ch);
+    }
+    else {
+      obj_to_char(obj, ch);       /* bread   */
+    }
+    if (!obj) {
+      send_to_char("Game error loading object. File a bug.\n\r", ch);
+    }
+    else {
+      obj = read_object(r_num, REAL);
+    }
+    if (!obj) {
+      send_to_char("Game error loading object. File a bug.\n\r", ch);
+    }
+    else {
+      obj_to_char(obj, ch);       /* bread   */
+    }
```

Comment:
This seems a little awkward, putting the `!obj` test in front of every line. What about extracting this function instead?

```
void load_obj_for_char(int obj_vnum, struct char_data *ch) {
  struct obj_data *obj;
  int r_num = real_object(obj_vnum);
  if (r_num < 0) {
    send_to_char("Game error loading object. File a bug.\n\r", ch);
    return;
  }
  obj = read_object(r_num, REAL);
  if (!obj) {
    send_to_char("Game error loading object. File a bug.\n\r", ch);
    return;
  }
  obj_to_char(obj, ch);
}
```

and then you can just assign the breads with:

```
load_obj_for_char(12, ch);
load_obj_for_char(12, ch);
```

Or, perhaps `load_obj_for_char` could take a quantity too.